test-fixtures/urn/f02/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "description": "F02. 856 without urn, and is non-legal deposit; Adds 856 with urn (https)",
+  "comment": "should we actually add non-resolvable urns?",
```

Comment:
I think since URN is based on ISBN that is permanent identifier in a sense is ok to add since theoretically there may be a point in future where the URN will resolve?